### ISSUE TITLE
Cannot install libcvd on OSX 10.9.1

### ISSUE THREAD

**Issue by headart** [randvoorhies/homebrew-rosten#2]
When I try to instal libCVD I am getting the following error and do not know how to overcome it.

```
Error: Operation already in progress for jpeg
Another active Homebrew process is already using jpeg.
Please wait for it to finish or terminate it to continue.
```

There are no other homebrew packages running (as far as I can tell).

**Comment by randvoorhies** [comment]
This sounds like it may be an issue with your system. Can you try running `brew doctor` to see if there are any outstanding issues?

**Comment by headart** [comment]
Yeah I've tried running `brew doctor` and it says that it is fine to build but still says that jpeg is busy.

This is the output of the build when I run `brew install libcvd` without options: https://gist.github.com/8912267

I notice that in the output it mentions that `NEON support is not enabled` with xcode. Could this have something to do with it?

**Comment by gugahoi** [comment]
Not working for me either on 10.9.4 clean system.
